src/daemon/rpc.rs

```diff
@@ -73,9 +81,307 @@ impl rpc::VarlinkInterface for Server {
             Err(e) => call.reply_invalid_parameter(e),
         }
     }
+
+    fn monitor(&self, call: &mut dyn rpc::Call_Monitor) -> varlink::Result<()> {
+        debug!("starting monitor worker loop");
+        if !call.wants_more() {
+            return call.reply_invalid_parameter("wants_more".into());
+        }
+
+        let (tx, rx) = chan::unbounded();
+        self.build_tx
+            .send(LoopHandlerEvent::NewListener(tx))
+            .map_err(|_| varlink::error::ErrorKind::Server)?;
+
+        call.set_continues(true);
+        for event in rx {
+            debug!("event for varlink"; "event" => format!("{:#?}", &event));
+            match event.try_into() {
+                Ok(ev) => {
+                    debug!("translated"; "varlink event" => format!("{:#?}", &ev));
+                    call.reply(ev)
+                }
+                Err(e) => {
+                    debug!("receiving event"; "error" => format!("{:#?}", &e));
+                    call.reply_invalid_parameter(e.to_string())
+                }
+            }?;
+        }
+        debug!("worker loop done for monitor");
```

Comment:
This debug message seems likely to cause spam. It's probably useful for building this feature, but I suggest removing it before this PR is merged.